templates/rtinfo/rtinfo.py

```diff
@@ -0,0 +1,52 @@
+from jumpscale import j
+from zerorobot.template.base import TemplateBase
+from zerorobot.template.state import StateCheckError
+
+NODE_CLIENT = "local"
+
+
+class Rtinfo(TemplateBase):
+    version = "0.0.1"
+    template_name = "rtinfo"
+
+    def __init__(self, name, guid=None, data=None):
+        super().__init__(name=name, guid=guid, data=data)
+        self.add_delete_callback(self.uninstall)
+
+    def validate(self):
+        # validate that rtinfo client is running when installed
+        self._monitor()
```

Comment:
that's not what the validate method if for. In here you can check the validity of the data in the schema.